sdk/trace/batch_span_processor.go

```diff
@@ -18,8 +18,10 @@ import (
 
 // Defaults for BatchSpanProcessorOptions.
 const (
-	DefaultMaxQueueSize       = 2048
-	DefaultScheduleDelay      = 5000
+	DefaultMaxQueueSize = 2048
+	// DefaultScheduleDelay is 5000 msec. It must be multiplied by time.Millisecond when passed to WithBatchTimeout.
```

Comment:
IMHO this doc should describe what the config does.
Giving the value provides little benefit, since it's readable right below (in godoc too).


```suggestion
	// DefaultScheduleDelay is the delay interval between two consecutive exports, in milliseconds.
```

Same for the other one.